src/server/unit_sao.cpp

```diff
@@ -273,15 +277,26 @@ std::string UnitSAO::generateUpdateAttachmentCommand() const
 }
 
 std::string UnitSAO::generateUpdateBonePositionCommand(
-		const std::string &bone, const v3f &position, const v3f &rotation)
+		const std::string &bone, const BoneOverride *override)
 {
 	std::ostringstream os(std::ios::binary);
 	// command
 	writeU8(os, AO_CMD_SET_BONE_POSITION);
 	// parameters
 	os << serializeString16(bone);
-	writeV3F32(os, position);
-	writeV3F32(os, rotation);
+	writeV3F32(os, override->position.vector);
+	v3f euler_rot;
+	override->rotation.next.toEuler(euler_rot);
+	writeV3F32(os, euler_rot * core::RADTODEG);
+	writeV3F32(os, override->scale.vector);
+	writeF32(os, override->position.interpolation_duration);
+	writeF32(os, override->rotation.interpolation_duration);
+	writeF32(os, override->scale.interpolation_duration);
+	// clang-format off
+	writeU8(os, override->position.absolute * 1
+	          + override->rotation.absolute * 2
+	          + override->scale.absolute    * 4);
```

Comment:
```suggestion
	writeU8(os, (override->position.absolute & 1) << 0
	          | (override->rotation.absolute & 1) << 1
	          | (override->scale.absolute & 1) << 2);
```
In the same style as `PlayerControl`. This 1. ensures proper values and 2. Obvious bit positions.